article/app/controllers/ArticleController.scala

```diff
@@ -80,14 +81,19 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
 
   }
 
-  private def render(path: String, page: PageWithStoryPackage)(implicit request: RequestHeader) = page match {
+  private def render(path: String, page: PageWithStoryPackage, pageNo: Option[Int])(implicit request: RequestHeader) = page match {
     case blog: LiveBlogPage =>
       if (request.isAmp) {
         MovedPermanently(path)
       } else {
-        val htmlResponse = () => views.html.liveBlog(blog)
-        val jsonResponse = () => views.html.fragments.liveBlogBody(blog)
-        renderFormat(htmlResponse, jsonResponse, blog, Switches.all)
+        val blocks = BlocksFor(blog.article.content.blocks, pageNo)
+        blocks match {
+          case Some(blocks) =>
+            val htmlResponse = () => views.html.liveBlog (blog, blocks)
+            val jsonResponse = () => views.html.fragments.liveBlogBody (blog, blocks)
+            renderFormat (htmlResponse, jsonResponse, blog, Switches.all)
```

Comment:
Minor one but we don't normally leave a space between fragment and passing parameters.